calcom/cal.com
# .env for root URL behavior+plausible #6193

## What does this PR do?

Adds 2 .env variables for Self Hosted Plausible URL, as well as root url to redirect users if they access the booking url without a session, this could be set to a particular users booking page, or to a landing page.

Fixes #6193 

**Environment**: Staging(main branch) / Production

## Type of change

- [✓ ] New feature (non-breaking change which adds functionality)
- [ ✓] This change requires a documentation update

## How should this be tested?

Configure the two new .env variables:

PLAUSIBLE_URL
ROOT_REDIR_URL

- You can see that traffic now sends to the self hosted Plausible URL, though you must include the full URL address, including the .js file.
- If you access the main WEBAPP_URL now without a session (incognito works), you should see you get redirected to the specified site.

## Checklist
